src/images/index.js

```diff
@@ -1,20 +1,22 @@
-import iconCoin from './icons/item-coins.svg';
-import iconFuel from './icons/item-fuel.svg';
-import iconGold from './icons/item-gold.svg';
-import iconIron from './icons/item-iron.svg';
-import iconEnergy from './icons/item-energy.svg';
-import imageWindpower from './buildings/windpower-plant.svg';
-import imageIronFoundry from './buildings/iron-foundry.svg';
-import imageOilRefinery from './buildings/oil-refinery.svg';
-import imageMoneyFactory from './buildings/money-factory.svg';
-import imageBackgroundMain from './background/background_main.jpg';
-import imageBackgroundManual from './background/background_manual.jpg';
+const iconCoin = 'https://smulski.de/capstone/images/icons/item-coins.svg';
+const iconFuel = 'https://smulski.de/capstone/images/icons/item-fuel.svg';
+const iconGold = 'https://smulski.de/capstone/images/icons/item-gold.svg';
+const iconIron = 'https://smulski.de/capstone/images/icons/item-iron.svg';
+const iconEnergy = 'https://smulski.de/capstone/images/icons/item-energy.svg';
+const imageWindpower = 'https://smulski.de/capstone/images/buildings/windpower-plant.svg';
+const imageIronFoundry = 'https://smulski.de/capstone/images/buildings/iron-foundry.svg';
+const imageOilRefinery = 'https://smulski.de/capstone/images/buildings/oil-refinery.svg';
+const imageMoneyFactory = 'https://smulski.de/capstone/images/buildings/money-factory.svg';
+const imageBackgroundMain = 'https://smulski.de/capstone/images/background/background_main.jpg';
+const imageBackgroundNav = 'https://smulski.de/capstone/images/background/background_nav.jpg';
+const imageBackgroundManual = 'https://smulski.de/capstone/images/background/background_manual.jpg';
```

Comment:
Why do you outsource the images now?

Reply:
I did a few tests with Stephan and it turned out that when the images come from external, he processes everything faster than when he pulls it from local, that was a performance jump of about 20% - 30%